docs/source/getting_started.rst

```diff
@@ -21,25 +21,48 @@ Installation
 ------------
 
 The easiest and fastest way to get the package up and running is to
-install poliastro using `conda <http://conda.io>`_::
+install poliastro using `conda <https://conda.io/docs/>`_::
 
   $ conda install poliastro --channel conda-forge
 
-You can also `install poliastro from PyPI`_ using pip, given that you already
-have all the requirements::
+.. note::
 
+    We encourage users to use conda and the
+    `conda-forge <https://conda-forge.org/>`_ packages for convenience,
+    especially when developing on Windows.
+
+If the installation fails for any reason, please open an issue in the
+`issue tracker`_.
+
+Alternative installation methods
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you don't want to use conda you can `install poliastro from PyPI`_ on Linux
+and OS X using pip::
+
+  $ pip install numpy  # Run this one first!
+  $ pip install scipy  # If this does not work, use conda
   $ pip install poliastro
 
-You can also `download poliastro source from GitHub`_ and type::
+If your platform is supported you can also install the accelerated version of
+poliastro using pip::
+
+  $ pip install poliastro[fast]
+
+However keep in mind that conda does this automatically on all platforms.
 
-  $ pip install .
+Finally, you can also install the latest development version of poliastro
+`directly from GitHub`_::
 
-Development installations are also supported thanks to setuptools::
+  $ pip install https://github.com/poliastro/poliastro/archive/master.zip
 
-  $ pip install -e .
```

Comment:
Good point. This should be part of a developer guide (see https://github.com/poliastro/poliastro/issues/176) but for the moment I agree to move it to `Contributing`.